lib/functions.php

```diff
@@ -143,11 +143,11 @@ function normalize(string $uri): string
     if (null !== $parts['scheme']) {
         $parts['scheme'] = strtolower($parts['scheme']);
         $defaultPorts = [
-            'http' => '80',
-            'https' => '443',
+            'http' => 80,
+            'https' => 443,
```

Comment:
Note:
`parse_url` always returns `$parts['port']` as an `int` (if it exists).
So we should do strict comparison below at line 150, and compare integers.
`phpstan` v2 wants strict `===` comparisons:
```
 ------ --------------------------------------------- 
  Line   lib/functions.php                            
 ------ --------------------------------------------- 
  150    Loose comparison via "==" is not allowed.    
         🪪  equal.notAllowed                          
         💡 Use strict comparison via "===" instead.   
 ------ --------------------------------------------- 
```